napari/_vispy/_tests/test_vispy_image_layer.py

```diff
@@ -144,6 +146,11 @@ def test_transforming_child_node(
     im_layer, translate, exp_translate, rotate, exp_rotate
 ):
     layer = VispyImageLayer(im_layer)
+
+    overlay = create_vispy_overlay(BoundingBoxOverlay(), layer=im_layer)
+    overlay.node.parent = layer.node
```

Comment:
Maybe some check that `layer.node.children[0]` is `overlay.node`?

Now it looks confusing when reading this test code. It should increase readability.